src/pages/landing.tsx

```diff
@@ -23,102 +23,226 @@ import { PrefetchPageLink } from "@/components/PrefetchPageLink"
 export function LandingPage() {
   const signIn = useSignIn()
   const isLoggedIn = useGlobalStore((s) => Boolean(s.session))
+
   return (
-    <div className="flex min-h-screen flex-col">
+    <div className="flex min-h-screen flex-col overflow-hidden">
       <Helmet>
-        <link rel="preconnect" href="https://img.shields.io" />
-        <link rel="dns-prefetch" href="https://img.shields.io" />
+        {/* Primary Meta Tags */}
+        <title>
+          TSCircuit - AI-Powered Electronics Design with Code | Build PCBs,
```

Comment:
tscircuit is always stylized fully lower case

```suggestion
          tscircuit - AI-Powered Electronics Design with Code | Build PCBs,
```